### ISSUE TITLE
Minecraft Bedrock 1.8 (Beta) crashes on launch on Arch Linux

### ISSUE THREAD

**Issue by NoahJelen** [minecraft-linux/mcpelauncher-manifest#16]
I am trying to launch the latest beta version of Minecraft Bedrock on Arch Linux and it crashes almost immediately after launching. Here is a screenshot of the game log.
![image](https://user-images.githubusercontent.com/29718341/46813142-fd7dca00-cd2a-11e8-9fa0-62d03fabb824.png)

**Comment by stayandwait** [comment]
same thing happens here, I am using the same version

**Comment by MCMrARM** [comment]
You must use the 1.8 branch: https://github.com/minecraft-linux/mcpelauncher-manifest/tree/1.8.0.8

**Comment by Mhowser** [comment]
What is the output of `which mcpelauncher-client` in a terminal?

**Comment by samdejong86** [comment]
I was able to get help from mrarm on the discord server.

Here's what `which mcpelauncher-client` output:
`/usr/local/bin/mcpelauncher-client`

instead of 
`/usr/bin/mcpelauncher-client`


This is because before I installed using the apt repository, I had installed from source. The apt repo launcher was grabbing the obsolete client, which couldn't run MCPE 1.8

The solution to this was to run

```
rm /usr/local/bin/msa-daemon
rm /usr/local/bin/msa-ui-qt
rm /usr/local/bin/msa-ui-gtk
rm /usr/local/bin/mcpelauncher-client
rm /usr/local/bin/mcpelauncher-server
rm -r /usr/local/share/mcpelauncher
```

as root.
